### DISCUSSION TITLE
Ring Intercom

### DISCUSSION THREAD

**Discussion by dpedr** [tsightler/ring-mqtt#791] [Support Q&A]
Hello, just a question, I know MQTT are available for devices with camera or alarms, my questions are: 
 -  Is available for Ring Intercom?.
 - Can I open or lock the door from HA with MQTT?
 
https://en-uk.ring.com/collections?category=ringintercom&fbclid=IwAR19zkg-SKv1B_1KMWT7bZgKzJd2U58cBubJ_LnRSe0pOjXKXzdp6BAQRF0

Thanks

**Comment by tsightler** [comment]
Supported Intercom features are documented in the wiki under the "Supported Devices and Features" section.
